### PR TITLE
Handle HSM errors appropriately

### PR DESCRIPTION
**Pull request by erikh360**
This change will only do the switch to SMS on the correct errors, time based failures will be dealt with in another endpoint.

I still need to do the translations.

### CODE REVIEW

File: changes/serializers.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ class ReceiveWhatsAppEventSerializer(serializers.Serializer):
     class StatusSerializer(serializers.Serializer):
 
         class ErrorSerializer(serializers.Serializer):
-            code = serializers.CharField(required=False)
+            code = serializers.IntegerField(required=False)
```
Comment (rudigiesler) [inline changes/serializers.py:53]: We'll probably need a migration for this?

**Review by rudigiesler — COMMENTED** [review]
Just one small comment, the rest of it looks good

File: changes/serializers.py (hunk 1 above)
Comment (erikh360) [reply changes/serializers.py:53]: Not for a serializer, do we?

Comment (rudigiesler) [reply changes/serializers.py:53]: Oh sorry, thought this was a model for some strange reason, 👍

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**